ui/src/app/dashboard/views/DiscoveriesList/DiscoveriesList.test.tsx

```diff
@@ -135,4 +135,32 @@ describe("DiscoveriesList", () => {
       .simulate("click");
     expect(wrapper.find("[data-test='delete-discovery']").exists()).toBe(true);
   });
+
+  it("can delete a discovery", () => {
+    const store = mockStore(state);
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter
+          initialEntries={[{ pathname: "/dashboard", key: "testKey" }]}
+        >
+          <DiscoveriesList />
+        </MemoryRouter>
+      </Provider>
+    );
+    // Open the action menu.
+    wrapper
+      .find("[data-test='row-menu'] button.p-contextual-menu__toggle")
+      .first()
+      .simulate("click");
+    // Click on the delete link.
+    wrapper
+      .find("button[data-test='delete-discovery-link']")
```

Comment:
Small thing but would be better to just target the `data-test`. If someone changes that button to a link in the future this test would fail unexpectedly.

Reply:
The issue in this case is that `[data-test='delete-discovery-link']` is an attribute on both the React component an the underlying button, so we need to target the element that has the click event.